docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
+containing many components.
+
+Tools like `Snapcraft`_ and `Juju`_ that use the concepts of parts to
```

Comment:
Does juju itself have the concept of parts, or just `charmcraft`? My understanding is that the operator framework doesn't use the parts lifecycle itself. (But I'm not confident enough to say for sure.)

Reply:
I was mixing them up, or just bundling everything together under the Juju name. :slightly_frowning_face: